lametro/views.py

```diff
@@ -81,10 +81,12 @@ def get_context_data(self, **kwargs):
             context['packet_url'] = None
 
         # Create list of related board reports, ordered by descending last_action_date.
+        # Thanks https://stackoverflow.com/a/2179053 for how to handle null last_action_date
         if context['legislation'].related_bills.all():
             all_related_bills = context['legislation'].related_bills.all().values('related_bill_identifier')
             related_bills = LAMetroBill.objects.filter(identifier__in=all_related_bills)
-            context['related_bills'] = sorted(related_bills, key=lambda bill: bill.last_action_date, reverse=True)
+            minimum_date = datetime.date(datetime.MINYEAR, 1, 1)
```

Comment:
you imported `MINYEAR` from `datetime`, so you can just access it as `MINYEAR` here.

Reply:
`minimum_date = datetime.date(MINYEAR, 1, 1)` like this?
that gets me `descriptor 'date' requires a 'datetime.datetime' object but received a 'int'`

Reply:
although, i **think** this would still return an error with the comparison, as trying to compare datetime and date objects doesn't work. 

switching to a datetime got me an error about naive and aware objects, which i'm still parsing.

Reply:
you want the datetime object, as you've already arrived at! see https://github.com/datamade/la-metro-councilmatic/pull/390/files#r248859862 for the comparison problemo.